test/Datadog.Trace.Security.IntegrationTests/AspNetCore5.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="AspNetCore5.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_0_OR_GREATER
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Security.IntegrationTests
+{
+    public class AspNetCore5 : AspNetBase, IDisposable
+    {
+        public AspNetCore5(ITestOutputHelper outputHelper)
+            : base("AspNetCore5", outputHelper)
+        {
+        }
+
+        [Theory]
+        [InlineData(true, true, HttpStatusCode.Forbidden)]
+        [InlineData(false, false, HttpStatusCode.OK)]
```

Comment:
Should we test with security enabled but blocking disabled?

Reply:
Yes, I prefer to do this after this PR is merged, logged as https://datadoghq.atlassian.net/browse/APPSEC-998

Reply:
Turned out to be easier than I thought as most things need for this are in place, see: 37c70b05f1aa0be83b995eca5193d6865db4e0a7